src/network/networkctl.c

```diff
@@ -119,7 +119,7 @@ static int check_netns_match(void) {
 
         if (id == 0) {
                 log_debug("systemd-networkd.service not running in a network namespace (?), skipping netns check.");
-                return 0;
+                goto pass;
```

Comment:
why use goto? insead you could have a else branch, below.

i mean, using goto is ok to use to jump downwards, but also: if we can avoid it easily, i'd still avoid it.